terraform/prod.vpc_peering.json

```diff
@@ -10,5 +10,17 @@
         "account_id": "650575079077",
         "vpc_id": "vpc-82c913e5",
         "subnet_cidr": "172.17.0.0/22"
+    },
+    {
+        "peer_name": "dit_trade_prod",
```

Comment:
The t in dit stands for trade, so this name is redundant. `dit_prod` (and `dit_dev`) is probably fine here.